src/ray/id.h

```diff
@@ -0,0 +1,71 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include <inttypes.h>
+
+#include <cstring>
+#include <string>
+
+#include "ray/util/visibility.h"
+
+#ifndef RAY_ID_H_
+#define RAY_ID_H_
+
+namespace ray {
+
+constexpr int64_t kUniqueIDSize = 20;
```

Comment:
How about putting this in `ray_config.h`?

Reply:
I think this is better kept here, since it is nothing that we will expose to the user.

Reply:
It can't really go into the class members of ray_config.h since then the compiler won't be able to optimize it out, but I'm putting all the compile time constants into a new file ray/constants.h now.